website/themes/uppy/source/css/_stats.scss

```
@@ -41,10 +41,13 @@ a.Stats-package {
 }
 
 .on-the-githubs {
```

```suggestion
.on-the-githubs {
  &:empty::before {
    content: "Loading...";
  }
```

(not sure `:empty` is the perfect selector as it might trigger if there's an error or something, I haven't checked, but it might work "well enough" for our use case)